javascript/javascript3/week1/WeatherApi/main.js

```diff
@@ -0,0 +1,87 @@
+const cityInput = document.getElementById('input-city')
+const btnSubmit = document.getElementById('btn-submit')
+const weatherDesUL = document.getElementById('weather-dec')
+const disTemp = document.querySelector('div .temp')
+const disCityDetail = document.querySelector('div .city-detail')
+const bgImg = document.querySelector('div .dis-part1')
+const disIcon = document.querySelector('div .weather-Icon')
+const disSunTime = document.getElementById('sun-time')
+const gmtDateTime = new Date().toUTCString();
+const apiKey = `aff81a037c13a860266157fd7cec79ed`
+let inputCityName = "copenhagen"
+btnSubmit.addEventListener('click', () => {
+    inputCityName= cityInput.value;
+    console.log(inputCityName)
+    if (!inputCityName) {
+        alert('enter city name')
+    }
+     display();
+});
+const display = () =>{
+    fetch(`https://api.openweathermap.org/data/2.5/weather?q=${inputCityName}&units=metric&appid=${apiKey}`)
+    .then(response => response.json())
+    .then(weatherData => {
+        const data = JSON.stringify(weatherData)
+        console.log(weatherData)
+        const { temp, feels_like, humidity, temp_max, temp_min } = weatherData.main;
+        const {speed} = weatherData.wind
+        console.log(speed)
+        const fahrenheit = (temp * 9) / 5 + 32;
+        const cityName = weatherData.name;
+        const {description, icon, main } = weatherData.weather[0];
+        const iconUrl = `http://openweathermap.org/img/wn/${icon}@2x.png`;
+        console.log(iconUrl);
+        const {sunrise, sunset } = weatherData.sys;
+        const sunriseGMT = new Date(sunrise * 1000);
+        const sunsetGMT = new Date(sunset * 1000);
+        disSunTime.innerHTML = `<h6>Sunrise: ${sunriseGMT.toUTCString()}</h6><h6>Sunset: ${sunsetGMT.toUTCString()}</h6>`
```

Comment:
instead of writing the html code into the string you should use `document.createElement`, it makes your code more readable

Reply:
yah, you are right.